pyzo/core/editor.py

```diff
@@ -259,6 +259,7 @@ def __init__(self, parent, **kwds):
         self.setWrap(bool(pyzo.config.view.wrap))
         self.setHighlightCurrentLine(pyzo.config.view.highlightCurrentLine)
         self.setLongLineIndicatorPosition(pyzo.config.view.edgeColumn)
+        self.setHighlightMatchingBracket(pyzo.config.view.highlightMatchingBracket)
```

Comment:
It makes pyzo fail during startup for me. Maybe you added it to your config somehow. It is easily resolved by adding the corresponding field to `resources/defaultConfig.ssdf`.